graphistry/embed_utils.py

```diff
@@ -0,0 +1,500 @@
+from matplotlib.font_manager import X11FontDirectories
```

Comment:
matplot lib is not a dep for us, and `X11FontDirectories` is a weird one to have